src/components/SideNavbar/SideNavbar.tsx

```diff
@@ -91,7 +90,7 @@ const NavItem: React.FC<NavItemProps> = ({ expanded = false, subitems, children,
 
   return (
     <SidebarNavItem>
-      <SidebarNavLink onClick={onClick} to={to} getProps={isActive} expanded={expanded} content={itemName}>
+      <SidebarNavLink onClick={onClick} to={to} expanded={expanded} content={itemName}>
```

Comment:
This isn't highlighted for home view when opening the app. Also when you use the Joystream logo to go home